.circleci/continue_config.yml

```diff
@@ -134,14 +134,17 @@ jobs:
   publish-snapshot-integration-spark:
     working_directory: ~/openlineage/integration/spark
     docker:
-      - image: circleci/openjdk:8-jdk
+      - image: cimg/openjdk:8
     steps:
       - *checkout_project_root
       - run: |
           # Get, then decode the GPG private key used to sign *.jar
           export ORG_GRADLE_PROJECT_signingKey=$(echo $GPG_SIGNING_KEY | base64 -d)
           export RELEASE_PASSWORD=$(echo $ARTIFACTORY_PASSWORD)
           export RELEASE_USERNAME=$(echo $ARTIFACTORY_USERNAME)
+          cd ../../client/java
+          ./gradlew --no-daemon publishToMavenLocal
+          cd -
```

Comment:
Rather than executing the build again, can we use `save_cache` and `restore_cache` to save the `.m2` directory like we do in the release jobs?

Reply:
Changed